src/Helper/Navigation/ConfigProvider.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\View\Helper\Navigation;
+
+use Laminas\ServiceManager\ConfigInterface;
+use Laminas\ServiceManager\Factory\InvokableFactory;
+use Laminas\ServiceManager\ServiceManager;
+
+/**
+ * @psalm-import-type FactoriesConfigurationType from ConfigInterface
+ * @psalm-import-type ServiceManagerConfiguration from ServiceManager
+ * @psalm-import-type ViewHelperConfigurationType from \Laminas\View\ConfigProvider
+ * @psalm-suppress DeprecatedClass
+ */
+final class ConfigProvider
```

Comment:
Components only have exactly **one** `ConfigProvider`.
If this component does also provide view-helpers, it has to be added to `Laminas\View\ConfigProvider`.